### PR TITLE
Book summaries

### PR DESCRIPTION
**Pull request by kbjarkefur**
I added this to the repo. I think it is a good place to save it for future use, and we can version control it and review it in PRs rather then back and forth in word or slack

### CODE REVIEW

File: book-blurbs/drip-summary-50-words.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+_The Development Research in Practice_ handbook is the quintessential desk reference
+for empirical researchers, policymakers, managers, and students. 
+It provides an introduction to modern, transparent, and ethical research practices involving data.
```
Comment (kbjarkefur) [inline book-blurbs/drip-summary-50-words.md:3]:
```suggestion
It provides an introduction to modern, transparent, and ethical research practices involving development data. 
```
I'd be more confident with this distinctions in the scoop

File: book-blurbs/drip-summary-300-words.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+and details special considerations for collecting high-quality survey data. 
+The fifth chapter dives into the process of cleaning and exploring research data. 
+The sixth chapter provides a guide to building analytical datasets, 
+writing data analysis code, and creating reproducible research outputs. 
+The seventh chapter addresses research publication, 
+including the preparation of reproducibility packages and publishing data. 
+The practical approach and narrative structure of these chapters are designed
+for both newcomers to empirical researchas well as experienced professionals. 
+The Development Research in Practice handbook is therefore a quintessential desk reference 
+for anyone who handles data as part of a research team, policymakers and managers who oversee such endeavors,
```
Comment (kbjarkefur) [inline book-blurbs/drip-summary-300-words.md:25]:
```suggestion
for anyone who handles development data as part of a research team, policymakers and managers who oversee such endeavors, 
```
We make the distinction in the beginning, but I'd be more comfortable if we make it clear we are abundantly clear that the scope is limited to development data

File: book-blurbs/drip-summary-300-words.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+_Development Research in Practice_ is intended to teach all users of development data 
+how to handle data effectively, efficiently, and ethically. 
+The Handbook leads the reader through a complete empirical research project,
```
Comment (kbjarkefur) [inline book-blurbs/drip-summary-300-words.md:3]:
```suggestion
The handbook leads the reader through a complete empirical research project,
```

File: book-blurbs/drip-summary-300-words.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+including research design, data mapping, and implementation of randomized study designs. 
+The fourth chapter covers the data acquisition process for primary and secondary data,
+and details special considerations for collecting high-quality survey data. 
+The fifth chapter dives into the process of cleaning and exploring research data. 
+The sixth chapter provides a guide to building analytical datasets, 
+writing data analysis code, and creating reproducible research outputs. 
+The seventh chapter addresses research publication, 
+including the preparation of reproducibility packages and publishing data. 
+The practical approach and narrative structure of these chapters are designed
+for both newcomers to empirical researchas well as experienced professionals.
```
Comment (kbjarkefur) [inline book-blurbs/drip-summary-300-words.md:23]:
```suggestion
for both newcomers to empirical research as well as experienced professionals. 
```

File: material-about-this-book/drip-summary-300-words.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+_Development Research in Practice_ is intended to teach all users of development data 
+how to handle data effectively, efficiently, and ethically. 
+The Handbook leads the reader through a complete empirical research project,
```
Comment (kbjarkefur) [inline material-about-this-book/drip-summary-300-words.md:3]:
```suggestion
The handbook leads the reader through a complete empirical research project,
```

File: material-about-this-book/drip-summary-300-words.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+and details special considerations for collecting high-quality survey data. 
+The fifth chapter dives into the process of cleaning and exploring research data. 
+The sixth chapter provides a guide to building analytical datasets, 
+writing data analysis code, and creating reproducible research outputs. 
+The seventh chapter addresses research publication, 
+including the preparation of reproducibility packages and publishing data. 
+The practical approach and narrative structure of these chapters are designed
+for both newcomers to empirical research as well as experienced professionals. 
+The Development Research in Practice handbook is therefore a quintessential desk reference 
+for anyone who handles data as part of a research team, policymakers and managers who oversee such endeavors,
```
Comment (kbjarkefur) [inline material-about-this-book/drip-summary-300-words.md:25]:
```suggestion
for anyone who handles development data as part of a research team, policymakers and managers who oversee such endeavors, 
```
We make the distinction in the beginning, but I'd be more comfortable if we make it clear we are abundantly clear that the scope is limited to development data

File: material-about-this-book/drip-summary-50-words.md [hunk 7]
```diff
@@ -0,0 +1,4 @@
+_The Development Research in Practice_ handbook is the quintessential desk reference
+for empirical researchers, policymakers, managers, and students. 
+It provides an introduction to modern, transparent, and ethical research practices involving data.
```
Comment (kbjarkefur) [inline material-about-this-book/drip-summary-50-words.md:3]:
```suggestion
It provides an introduction to modern, transparent, and ethical research practices involving development data. 
```

File: material-about-this-book/drip-summary-300-words.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
+_Development Research in Practice_ is intended to teach all users of development data
+how to handle data effectively, efficiently, and ethically.
+The Handbook leads the reader through a complete empirical research project,
+providing links to continuously-updated resources on the DIME Wiki
+as well as illustrative examples from the Demand for Safe Spaces study conducted in Rio de Janeiro.
+The premise is that standardizing and simplifying data tasks
```
Comment (luizaandrade) [inline material-about-this-book/drip-summary-300-words.md:6]:
```suggestion
Its premise is that standardizing and simplifying data tasks
```

File: material-about-this-book/drip-summary-300-words.md (hunk 5 above)
Comment (mariaruth) [reply material-about-this-book/drip-summary-300-words.md:3]: should be capitalized because is reference to this particular Handbook

Comment (kbjarkefur) [reply material-about-this-book/drip-summary-300-words.md:3]: ok.

**Review by mariaruth — COMMENTED** [review]
other than the one change i didn't accept this seems fine to me

**Review by mariaruth — APPROVED** [review]

**Pull request merged**